scripts/check_nightly.py

```diff
@@ -39,41 +34,26 @@
 
 def expensive_tests_in_file(path: pathlib.Path) -> typing.Iterable[str]:
     """Yields names of expensive tests found in given Rust file.
-
-    An expensive test is a function annotated with `test` and a conditional
-    `ignore` attributes, specifically:
-
-        #[test]
-        #[cfg_attr(not(feature = "expensive_tests"), ignore)]
-        fn test_slow() {
-            // ...
-        }
-
-    Note that anything more complex in the `cfg_attr` will cause the function
-    not to recognize the test.
-
     Args:
         path: Path to the Rust source file.
     Yields:
-        Names of functions defining expensive tests (e.g. `test_slow` in example
-        above).
+        Names of functions defining expensive tests (e.g. `ultra_slow_test_gc_random_large`
+        in example above).
     """
     with open(path) as rd:
-        is_expensive = False
         is_test = False
         for line in rd:
             line = line.strip()
             if not line:
                 pass
             elif line.startswith("#"):
                 is_test = is_test or line == TEST_DIRECTIVE
-            elif line.startswith("fn ultra_slow_test_"):
```

Comment:
optional nit: since `EXPENSIVE_DIRECTIVE` is used exactly in one place and `ultra_slow_test_` isn't a directive it may be cleaner to keep string here as it was and remove `EXPENSIVE_DIRECTIVE` altogether.